src/devices/machine/mc68681.cpp

```diff
@@ -624,6 +691,81 @@ WRITE8_MEMBER( sc28c94_device::write )
 	}
 }
 
+WRITE8_MEMBER( xr68c681_device::write )
+{
+	if (offset == 0x02) /* CRA */
+		switch(data >> 4)
```

Comment:
Spacing is off here - you've got `if (...` (space) and `switch(...` (no space).  Both are control statements, so they should both have the space.